extensions/amp-story/0.1/media-tasks.js

```diff
@@ -216,6 +219,9 @@ export class MediaTask {
  * Plays the specified media element.
  */
 export class PlayTask extends MediaTask {
+  /**
+   * @public
```

Comment:
Don't think you need the @public?  Might be safer to just as a comment? (Here and in other tasks)

Reply:
The linter requires us to put something here... Wasn't sure about what to use. I'm okay with using a comment instead, but I'll have to be creative haha, there's not much to say about these constructors. What do you think is better?

Reply:
I just merged this since other PRs were blocked on JSDoc errors. I'll resolve this comment in a following PR depending on what you think is best :))